src/components/App.js

```diff
@@ -75,10 +76,19 @@ class App extends Component {
         </div>
         {
           !this.props.auth.loading && this.state.posts.length === 0? (
-            <div className="uk-align-center">
+            <div className="uk-align-center uk-text-center">
+              <img src="https://www.dailydot.com/wp-content/uploads/e52/31/87610fa1a0ae891d.png" alt="No Feed Pic"  height="300px" width="300px"/>
```

Comment:
Don't use hosted images, you can put them in the static files ad then use them.

Reply:
okay sir!